qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -246,8 +251,50 @@ def selection(self, html=False):
             raise browsertab.UnsupportedOperationError
         return self._widget.selectedText()
 
+    def _follow_selected_cb(self, js_elem, tab=False, repeat_limit=0):
+        """Callback for javascript which clicks the selected element.
+
+        Used for follow_selected below
+
+        Args:
+            js_elems: The elements serialized from javascript.
+            tab: Open in a new tab or not.
+            repeat_limit: The number of times we have to retry this function."""
+        if js_elem is None:
+            # Try to repeat this request
+            # TODO when on Qt5.9, don't repeat at all, and use selectionChanged
+            # https://bugreports.qt.io/browse/QTBUG-53134
+            if repeat_limit:
+                QTimer.singleShot(
```

Comment:
actually it seems like this hasn't fixed it, I'm going to keep messing around, and hopefully I can find it... I wish I could reproduce this on my own machine (sigh)